chart/values.yaml

```diff
@@ -721,7 +721,7 @@ webserver:
       from: []
       # Ports for webserver NetworkPolicy ingress (if `from` is set)
       ports:
-        - port: airflow-ui
+        - port: "{{ .Values.ports.airflowUI }}"
```

Comment:
This needs to be added in https://github.com/apache/airflow/blob/main/chart/UPDATING.rst

Reply:
Network Policy parameter for webserver  and flower  UI moved away from named port to port-number to avoid issues with openshift - is this okay

Reply:
updated rst file - pls verify @kaxil and @jedcunningham